src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -260,38 +257,32 @@ const PersonalInfoList = ({
       />
     ) : null;
 
-  const note = isFacStaff ? (
-    <Typography align="left" className="note">
-      NOTE:
-      <ul>
-        <li>
-          To prevent your picture from displaying click{' '}
-          <a href="https://360.gordon.edu/myprofile"> here</a>.
-        </li>
-        <li>
-          To update your data contact <a href="mailto: hr@gordon.edu">Human Resources</a> (x4828).
-        </li>
-      </ul>
-    </Typography>
-  ) : isStudent ? (
-    <Typography align="left" className="note">
-      NOTE:
-      <ul>
-        <li>
-          To prevent your picture or your cell phone number from displaying, click{' '}
-          <a href="https://360.gordon.edu/myprofile">here</a>.
-        </li>
-        <li>
-          To update your On Campus Address, contact <a href="mailto: housing@gordon.edu">Housing</a>
-          (x4263).
-        </li>
-        <li>
-          For all other changes or to partially/fully prevent your data from displaying, contact the{' '}
-          <a href="mailto: registrar@gordon.edu">Registrar's Office</a> (x4242).
-        </li>
-      </ul>
-    </Typography>
-  ) : null;
+  const note =
+    myProf && isFacStaff ? (
+      <Typography align="left" className="note">
+        NOTE:
+        <ul>
+          <li>
+            To update your data, please contact <a href="mailto: hr@gordon.edu">Human Resources</a>{' '}
+            (x4828).
+          </li>
+        </ul>
+      </Typography>
+    ) : myProf && isStudent ? (
```

Comment:
```suggestion
    ) : isStudent ? (
```
The check here for myProf is unneccesary since we can't get to it unless myProf was already true on line 261 (though you could keep it just to make it clear too).

Reply:
myProf is necessary. This makes NOTES only appear on My Profile page.

For example, when I search Kevin on People Search...

with myProf:
![Screen Shot 2021-06-18 at 9 19 09 AM](https://user-images.githubusercontent.com/70544403/122567404-a02b9200-d016-11eb-818c-37219e263215.png)

without myProf:
![Screen Shot 2021-06-18 at 9 19 20 AM (2)](https://user-images.githubusercontent.com/70544403/122567362-94d86680-d016-11eb-8098-b5108dec1dd1.png)

same thing on line number 261.
These two lines are the main changes of this issue.

Reply:
Yes, sorry, that was incorrect. I will make a suggestion that works correctly and is a bit cleaner.